### ISSUE TITLE
Failed to find Storybook presets (Windows)

### ISSUE THREAD

**Issue by omarbelkhodja** [literalpie/storybook-framework-qwik#19]
I tried to use this extension, but I'm facing the following failure. Any idea ?

```powershell
E:\git\qwik-test\qwik-app>npm run storybook

> storybook
> storybook dev -p 6006

@storybook/cli v7.0.0-beta.19

(node:21796) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
WARN   Failed to load preset: "storybook-framework-qwik\\preset"
ERR! Error: Cannot find module 'storybook-framework-qwik\preset'
ERR! Require stack:
ERR! - E:\git\qwik-test\qwik-app\node_modules\@storybook\core-common\dist\index.js
ERR! - E:\git\qwik-test\qwik-app\node_modules\@storybook\telemetry\dist\index.js
ERR! - E:\git\qwik-test\qwik-app\node_modules\@storybook\cli\dist\generate.js
ERR! - E:\git\qwik-test\qwik-app\node_modules\@storybook\cli\bin\index.js
ERR! - E:\git\qwik-test\qwik-app\node_modules\storybook\index.js
ERR!     at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
ERR!     at Module._load (node:internal/modules/cjs/loader:841:27)
ERR!     at Module.require (node:internal/modules/cjs/loader:1061:19)
ERR!     at require (node:internal/modules/cjs/helpers:103:18)
ERR!     at interopRequireDefault (E:\git\qwik-test\qwik-app\node_modules\@storybook\core-common\dist\index.js:6:21)
ERR!     at getContent (E:\git\qwik-test\qwik-app\node_modules\@storybook\core-common\dist\index.js:10:164)
ERR!     at loadPreset (E:\git\qwik-test\qwik-app\node_modules\@storybook\core-common\dist\index.js:10:349)
ERR!     at E:\git\qwik-test\qwik-app\node_modules\@storybook\core-common\dist\index.js:12:409
ERR!     at Array.map (<anonymous>)
ERR!     at loadPresets (E:\git\qwik-test\qwik-app\node_modules\@storybook\core-common\dist\index.js:12:391)
ERR!  Error: Cannot find module 'storybook-framework-qwik\preset'
ERR! Require stack:
ERR! - E:\git\qwik-test\qwik-app\node_modules\@storybook\core-common\dist\index.js
ERR! - E:\git\qwik-test\qwik-app\node_modules\@storybook\telemetry\dist\index.js
ERR! - E:\git\qwik-test\qwik-app\node_modules\@storybook\cli\dist\generate.js
ERR! - E:\git\qwik-test\qwik-app\node_modules\@storybook\cli\bin\index.js
ERR! - E:\git\qwik-test\qwik-app\node_modules\storybook\index.js
ERR!     at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
ERR!     at Module._load (node:internal/modules/cjs/loader:841:27)
ERR!     at Module.require (node:internal/modules/cjs/loader:1061:19)
ERR!     at require (node:internal/modules/cjs/helpers:103:18)
ERR!     at interopRequireDefault (E:\git\qwik-test\qwik-app\node_modules\@storybook\core-common\dist\index.js:6:21)
ERR!     at getContent (E:\git\qwik-test\qwik-app\node_modules\@storybook\core-common\dist\index.js:10:164)
ERR!     at loadPreset (E:\git\qwik-test\qwik-app\node_modules\@storybook\core-common\dist\index.js:10:349)
ERR!     at E:\git\qwik-test\qwik-app\node_modules\@storybook\core-common\dist\index.js:12:409
ERR!     at Array.map (<anonymous>)
ERR!     at loadPresets (E:\git\qwik-test\qwik-app\node_modules\@storybook\core-common\dist\index.js:12:391) {
ERR!   code: 'MODULE_NOT_FOUND',
ERR!   requireStack: [
ERR!     'E:\\git\\qwik-test\\qwik-app\\node_modules\\@storybook\\core-common\\dist\\index.js',
ERR!     'E:\\git\\qwik-test\\qwik-app\\node_modules\\@storybook\\telemetry\\dist\\index.js',
ERR!     'E:\\git\\qwik-test\\qwik-app\\node_modules\\@storybook\\cli\\dist\\generate.js',
ERR!     'E:\\git\\qwik-test\\qwik-app\\node_modules\\@storybook\\cli\\bin\\index.js',
ERR!     'E:\\git\\qwik-test\\qwik-app\\node_modules\\storybook\\index.js'
ERR!   ]
ERR! }
ERR! TypeError: Cannot destructure property 'renderer' of '(intermediate value)' as it is undefined.
ERR!     at buildDevStandalone (E:\git\qwik-test\qwik-app\node_modules\@storybook\core-server\dist\index.js:33:1795)
ERR!     at async withTelemetry (E:\git\qwik-test\qwik-app\node_modules\@storybook\core-server\dist\index.js:33:5533)
ERR!     at async dev (E:\git\qwik-test\qwik-app\node_modules\@storybook\cli\dist\generate.js:441:440)
ERR!  TypeError: Cannot destructure property 'renderer' of '(intermediate value)' as it is undefined.
ERR!     at buildDevStandalone (E:\git\qwik-test\qwik-app\node_modules\@storybook\core-server\dist\index.js:33:1795)
ERR!     at async withTelemetry (E:\git\qwik-test\qwik-app\node_modules\@storybook\core-server\dist\index.js:33:5533)
ERR!     at async dev (E:\git\qwik-test\qwik-app\node_modules\@storybook\cli\dist\generate.js:441:440)

WARN Broken build, fix the error above.
WARN You may need to refresh the browser.
```

**Comment by literalpie** [comment]
hmm that's strange. Can you double check that `storybook-framework-qwik/dist/preset.js` exists in your node modules? Also, can you run `npc storybook info` and tell me the result?

**Comment by literalpie** [comment]
(or if you can send the whole repo with the error, that would be even better)

**Comment by omarbelkhodja** [comment]
```powershell
C:\qwik-app>npm run storybook.info

> storybook.info
> storybook info


Environment Info:

  System:
    OS: Windows 10 10.0.22621
    CPU: (12) x64 Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
  Binaries:
    Node: 18.12.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.18.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22621.963.0), Chromium (108.0.1462.54)
  npmPackages:
    @storybook/addon-essentials: ^7.0.0-beta.19 => 7.0.0-beta.19 
    @storybook/addon-interactions: ^7.0.0-beta.19 => 7.0.0-beta.19 
    @storybook/addon-links: ^7.0.0-beta.19 => 7.0.0-beta.19
    @storybook/blocks: ^7.0.0-beta.19 => 7.0.0-beta.19
    @storybook/html: ^7.0.0-beta.19 => 7.0.0-beta.19
    @storybook/html-vite: ^7.0.0-beta.19 => 7.0.0-beta.19
    @storybook/testing-library: ^0.0.13 => 0.0.13
```

**Comment by omarbelkhodja** [comment]
```powershell
Répertoire de C:\qwik-app\node_modules\storybook-framework-qwik\dist

06/01/2023  10:44 AM               737 preset.js
               1 fichier(s)              737 octets
               0 Rép(s)  221 581 684 736 octets libres
```

**Comment by omarbelkhodja** [comment]
I tried in a devcontainer (which means in a Linux environment). The issue then is not reproduced. So, it is typically related to Windows environment.

**Comment by literalpie** [comment]
Ah, I was wondering if that could be it. I'll look into this more. Thanks for the extra information!

**Comment by literalpie** [comment]
Hi, this should be fixed in the version I just released: `0.0.6`. Please let me know if this works for you!

**Comment by omarbelkhodja** [comment]
Great ! Tested and I confirm it's fixed. Thanks a lot !
